docs/cloud-adoption/operations/operational-fitness-review.md

```diff
@@ -0,0 +1,127 @@
+
+## Fundamentals review example
+
+Contoso is an online retailer that has recently added the sales of event tickets to their business. Contoso creates a new [e-commerce front-end on Azure](../../example-scenario/apps/ecommerce-scenario.md) to perform the following functions:
+
+- Host the tickets sales web site front end. Users sign in to view concert details, order tickets, check the status of ordered tickets, and view and show tickets to gain entry to the concert venue. Images and other static content are served to the user through a content delivery network (CDN).
+- Payments are made using a third-party payment-processing system.
+- Users input concert reviews after the event has occurred, and machine learning is applied to the reviews to determine concert-goer sentiment which is provided to the concert promoter.
+
+Contoso has recently sold tickets for several popular concerts but has received complaints from customers that it's difficult to view concert details and order tickets once they go on sale. Several customers have also complained that they are randomly logged out of the site. Contoso has analyzed the performance of the application code and determined that there are no bugs in any feature that was cause the performance degradations or issues with authentication.
+
+### Goals & Expected Outcomes
+
+Contoso has made a large investment in the business relationship with concert promoters as well as the development of the concert ticket sales site. Selling concerts is a "mission critical" function for Contoso, so they prioritize fixing the issues with the site as its highest priority.
+
+Contoso begins the fundamentals process by defining the service level objective (SLO) for the concert ticket sales application. The service level objective is comprised of the following objectives: 
+* recovery point objective (RPO), the maximum acceptable time period in which data might be lost during an incident. 
+* recovery time objective (RTO), the maximum acceptable time period for the service to be unavailable.
+
+The concert ticket sales application is very sensitive to data loss because user's ticket images are stored and would prevent a user being able to attend a concert they had paid for if the ticket image were unavailable. Contoso's customers are also very sensitive to the application being unavailable because the customers are not always able to discern between the application being down versus the site being available but too busy to serve them. In this case, a customer might feel that they are missing out on the opportunity to purchase tickets because other customers are able to buy them and the concert will soon be sold out.
+
+### Process
+
+The operational fitness review team meets on a regular basis. The team meets on a monthly cadence and reports status and metrics to senior leadership on a quarterly basis.
```

Comment:
General comment: At this point we seem to leave Contoso behind. Is it possible to connect this process back to the Contoso example?